src/ISLogger.cpp

```diff
@@ -678,57 +677,23 @@ float cISLogger::LogSizeAllMB()
 
 float cISLogger::LogSizeMB(uint32_t devSerialNo)
 {
-    return (m_devices.find(devSerialNo) == m_devices.end()) ? m_devices[devSerialNo]->LogSize() * 0.000001f : 0;
+    return (DeviceFind(devSerialNo) ? m_devices[devSerialNo]->LogSize() * 0.000001f : 0);
 }
 
 float cISLogger::FileSizeMB(uint32_t devSerialNo)
 {
-    return (m_devices.find(devSerialNo) == m_devices.end()) ? m_devices[devSerialNo]->FileSize() * 0.000001f : 0;
+    return (DeviceFind(devSerialNo) ? m_devices[devSerialNo]->FileSize() * 0.000001f : 0);
 }
 
 uint32_t cISLogger::FileCount(uint32_t devSerialNo)
 {
-    return (m_devices.find(devSerialNo) == m_devices.end()) ? m_devices[devSerialNo]->FileCount() : 0;
+    return (DeviceFind(devSerialNo) ? m_devices[devSerialNo]->FileCount() : 0);
 }
 
 std::string cISLogger::GetNewFileName(uint32_t devSerialNo, uint32_t fileCount, const char *suffix)
 {
-    return (m_devices.find(devSerialNo) == m_devices.end()) ? m_devices[devSerialNo]->GetNewFileName(devSerialNo, fileCount, suffix) : std::string("");
-}
-
-/*
-bool cISLogger::SetDevice(const ISDevice *info)
```

Comment:
I am not entirely sure why these were commented out.  I thought they were used elsewhere in the code; but clearly not.  Maybe I commented them out, thinking they were, but clearly aren't, so... yeah, let's delete the code.